buddyboss/buddyboss-platform
# PROD-697 Fix Changing group avatar also changing the user avatar on the header for time being

### All Submissions:


Fixes #3128 

### How to test the changes in this Pull Request:

1. Go to a group
2. Change the group avatar
3. after crop and success update
4. See the header user profile

### Proof Screenshots or Video
https://www.loom.com/share/6ee70fb53eeb4ca1a3929146b1ab5e75

### Other information:
